File: app/src/main/java/ir/ahfz/rentcar/io/network/model/MakeResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;


public class MakeResponse extends BaseResponse {

    private List<Make> makes = null;

    public List<Make> getMakes() {
        return makes;
    }

    public void setMakes(List<Make> makes) {
        this.makes = makes;
    }

    public static class Make {

        private int id;
        private String make;
        private String createdAt;
        private String updatedAt;

        public int getId() {
            return id;
        }

        public void setId(int id) {
            this.id = id;
        }

        public String getMake() {
            return make;
        }

        public void setMake(String make) {
            this.make = make;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

    }

}

File: app/src/main/java/ir/ahfz/rentcar/io/network/model/MakeResponse.kt
package ir.ahfz.rentcar.io.network.model

import ir.ahfz.rentcar.io.network.BaseResponse
/**
 * Data model for Make response
 */
data class MakeResponse(var makes: List<Make>? = null) : BaseResponse() {
    /**
     * Data model for Make
     */
    data class Make(
        var id: Int = 0,
        var make: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null
    )
}